multimodal/src/autogluon/multimodal/learners/semantic_segmentation.py

```diff
@@ -30,6 +144,16 @@ def get_peft_param_names_per_run(model, config):
             )
         return peft_param_names
 
+    def get_loss_func_per_run(self, config, mixup_active=None):
+        loss_func = get_loss_func(
+            problem_type=self._problem_type,
+            mixup_active=mixup_active,
```

Comment:
Remove mixup if not used.

Reply:
get_loss_func() requires the positional argument 'mixup_active'.